src/utils.js

```diff
@@ -137,13 +137,14 @@ const handleFormUpdate = function(e) {
             break;
         case 'number':
         case 'range':
+        case 'button':
             obj[id] = parseFloat(e.target.value);
             break;
         case 'select-one':
             obj[id] = parseInt(e.target.value, 10);
             break;
         case 'textarea':
-            obj[id] = e.target.value;
+            obj[id] = parseFloat(e.target.value);
```

Comment:
We use textareas for the graph Summary and Instructions - these strings shouldn't be parsed to a float.